<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FilterDialog</class>
 <widget class="QDialog" name="FilterDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>338</width>
    <height>199</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>平滑滤波器</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog
{
	background-color:rgb(65, 65, 65);
}
QDialog QLabel
{
	color:rgb(221, 221, 221);
	qproperty-alignment: AlignCenter;
}
QPushButton
{
	border:2px groove rgb(221, 221, 221);
	border-radius: 6px;
	color:rgb(221, 221, 221);
	padding: 2px 5px;
}
QPushButton:hover
{
	border:2px groove rgb(221, 221, 221);
	color: rgb(83, 83, 83);
	background-color: rgb(221, 221, 221);
}
QPushButton:pressed
{
	border:2px groove rgb(70, 160, 245);
	color: rgb(221, 221, 221);
	background-color: rgb(70, 160, 245);
}
QRadioButton
{
	color:rgb(221, 221, 221);
}</string>
  </property>
  <widget class="QLabel" name="typeLabel">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>50</y>
     <width>60</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lantinghei SC</family>
    </font>
   </property>
   <property name="text">
    <string>核类型：</string>
   </property>
  </widget>
  <widget class="QLabel" name="sizeLabel">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>100</y>
     <width>60</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lantinghei SC</family>
    </font>
   </property>
   <property name="text">
    <string>核大小：</string>
   </property>
  </widget>
  <widget class="QPushButton" name="conformButton">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>150</y>
     <width>61</width>
     <height>32</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lantinghei TC</family>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>确认</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelButton">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>150</y>
     <width>61</width>
     <height>32</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lantinghei TC</family>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="meanButton">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>47</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lantinghei SC</family>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>均值</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="gaussianButton">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>47</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lantinghei SC</family>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>高斯</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="medianButton">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>47</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lantinghei SC</family>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>中值</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="size3Button">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>97</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lantinghei SC</family>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>3x3</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="size4Button">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>97</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lantinghei SC</family>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>4x4</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="size5Button">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>97</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lantinghei SC</family>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>5x5</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
